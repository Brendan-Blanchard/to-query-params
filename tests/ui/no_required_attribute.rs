use to_query_params::QueryParams;

#[derive(QueryParams)]
struct Data {
    number: f64,
}

fn main() {}
