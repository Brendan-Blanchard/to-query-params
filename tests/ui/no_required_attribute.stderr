error: proc-macro derive panicked
 --> tests/ui/no_required_attribute.rs:3:10
  |
3 | #[derive(QueryParams)]
  |          ^^^^^^^^^^^
  |
  = help: message: Non-optional types must be marked with #[query(required)] attribute
